src/components/deck-gl-wrapper/mesh-layer/mesh-layer.ts

```diff
@@ -0,0 +1,192 @@
+/* eslint-disable @typescript-eslint/ban-types */
+import type { NumericArray } from "@math.gl/core";
```

Comment:
comment changed code. If you removed something, keep removed code commented.
If no changes, leave a comment above that the file has been moved from deck.gl without changes to handle the fork of Tile3DLayer

Reply:
No changes from my side, added appropriate comment